### DISCUSSION TITLE
4.0.0-beta.1版本的依赖中workspace没有替换为实际版本号

### DISCUSSION THREAD

**Discussion by wangqi060934** [youzan/vant#10974] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1433283/186921253-47ec8b2f-1102-444c-878f-d63e22db3f92.png)
使用4.0.0-bate.1版本的时候，发现vant的package.json中内部模块的依赖还是"workspace:*"，不是实际版本号，不确定这算不算个问题。

**Comment by chenjiahan** [comment, marked as answer]
这几个是 devDependencies，不影响使用，我们后续优化下
